build.gradle

```diff
@@ -47,6 +66,7 @@ buildscript {
 }
 
 apply plugin: 'kotlinx.team.infra'
+ext.compilerVersion = org.jetbrains.kotlin.config.KotlinCompilerVersion.VERSION
```

Comment:
This is for infra or for some other reason? How this version differs from one printed in `compilerVersion.gradle`?

Reply:
To make it visible from `compilerVersion.gradle` (FQN works only in root gradle file and imports are not transitive, yay)